internal/proto/networkfence.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.34.2
-// 	protoc        v3.19.6
+// 	protoc        v3.20.2
```

Comment:
this is an unrelated change, there should be a commit that updates the protoc version and updated generated files.